### DISCUSSION TITLE
静态网站显示问题

### DISCUSSION THREAD

**Discussion by gaohan-cmd** [1Panel-dev/1Panel#4566] [Q&A]
为什么静态网站index目录下放php后缀首页，点击访问就弹出下载框，无法正常显示首页，请问怎么解决？
![image](https://github.com/1Panel-dev/1Panel/assets/63499259/2073c790-88f2-4fe4-87bd-1331bef5be90)
![image](https://github.com/1Panel-dev/1Panel/assets/63499259/de3838f7-a250-4972-868c-95d0d3cdaf74)

**Comment by xiexiaosheng** [comment]
你应该安装PHP运行环境，静态适用于html网页
